runhouse/resources/packages/package.py

```diff
@@ -232,7 +233,7 @@ def _install(self, env: Union[str, "Env"] = None, cluster: "Cluster" = None):
         logger.info(f"Installing {str(self)} with method {self.install_method}.")
 
         if isinstance(self.install_target, InstallTarget):
-            if cluster:
+            if cluster and os.path.exists(self.install_target.local_path):
```

Comment:
let's add this check in `cluster.rsync` as well? especially if it's a public utility?